Northstar.Client/mod/scripts/vscripts/ui/menu_mode_select.nut

```diff
@@ -1,86 +1,552 @@
+untyped
 global function InitModesMenu
+global function NSSetModeCategory
+
+global enum eModeMenuModeCategory
+{
+	UNKNOWN = 0,
+	PVPVE   = 1,
+	PVE     = 2,
+	PVP     = 3,
+	FFA     = 4,
+	TITAN   = 5,
+	OTHER   = 6,
+	CUSTOM  = 7
+
+	SIZE
```

Comment:
What's the meaning of this?

Reply:
`SIZE` will always be 1 more than the last entry in the enum. This allows the enum to be modified without having to adapt [other code](https://github.com/R2Northstar/NorthstarMods/pull/624/files#diff-0acf05adff8e31dbe3df62eba236d9001887b39f34abd248ed449a26ea59322cR87-R94)